CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ Please enumerate all user-facing changes using format `<githib issue/pr number>:
 
 ## 1.0.0
 
+* [#262](https://github.com/kroxylicious/kroxylicious-junit5-extension/pull/262): Support embedding a 3.7 broker
```

Comment:
Can you stick a 0.8.1 heading in as well please.

Reply:
done